src/main.rs

```diff
@@ -504,8 +514,13 @@ async fn make_app(current_dir: &std::ffi::OsString) -> anyhow::Result<Command> {
             )
         )
         .subcommand(Command::new("dev-ui")
-            .about("Start the web UI development server with hot reloading (same as `cd ui && npm i && npm start`)")
+            .about("Start the web UI development server with hot reloading (same as `cd ui && npm i && npm run dev`). To instead create a production build, use the --release flag.")
```

Comment:
This helptext is getting pretty long. Can we keep it as it was and just change the `npm start` to `npm run dev`?

Reply:
[f3d1590](https://github.com/kinode-dao/kit/pull/112/commits/f3d15901d642319c59e048b2d0671f3170653801)